2020-11-11 11:06:14.418877: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
--------------------------------------------------------------------------
The library attempted to open the following supporting CUDA libraries,
but each of them failed.  CUDA-aware support is disabled.
libcuda.so.1: cannot open shared object file: No such file or directory
libcuda.dylib: cannot open shared object file: No such file or directory
/usr/lib64/libcuda.so.1: cannot open shared object file: No such file or directory
/usr/lib64/libcuda.dylib: cannot open shared object file: No such file or directory
If you are not interested in CUDA-aware support, then run with
--mca mpi_cuda_support 0 to suppress this message.  If you are interested
in CUDA-aware support, then try setting LD_LIBRARY_PATH to the location
of libcuda.so.1 to get passed this issue.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
A process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.

The process that invoked fork was:

  Local host:          [[24910,1],0] (PID 14317)

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
WARNING:tensorflow:From /pkg/suse12/software/TensorFlow/2.3.1-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/tensorflow/python/compat/v2_compat.py:96: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
WARNING:tensorflow:OMP_NUM_THREADS is no longer used by the default Keras config. To configure the number of threads, use tf.config.threading APIs.
2020-11-11 11:07:00.528937: W tensorflow/stream_executor/platform/default/dso_loader.cc:59] Could not load dynamic library 'libcuda.so.1'; dlerror: libcuda.so.1: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /pkg/suse12/software/TensorFlow/2.3.1-fosscuda-2019b-Python-3.7.4/lib:/pkg/suse12/software/snappy/1.1.7-GCCcore-8.3.0/lib64:/pkg/suse12/software/libpng/1.6.37-GCCcore-8.3.0/lib:/pkg/suse12/software/protobuf/3.10.0-GCCcore-8.3.0/lib64:/pkg/suse12/software/PCRE/8.43-GCCcore-8.3.0/lib:/pkg/suse12/software/nsync/1.24.0-GCCcore-8.3.0/lib64:/pkg/suse12/software/LMDB/0.9.24-GCCcore-8.3.0/lib:/pkg/suse12/software/libjpeg-turbo/2.0.3-GCCcore-8.3.0/lib64:/pkg/suse12/software/JsonCpp/1.9.3-GCCcore-8.3.0/lib64:/pkg/suse12/software/ICU/64.2-GCCcore-8.3.0/lib:/pkg/suse12/software/giflib/5.2.1-GCCcore-8.3.0/lib:/pkg/suse12/software/flatbuffers/1.12.0-GCCcore-8.3.0/lib64:/pkg/suse12/software/double-conversion/3.1.4-GCCcore-8.3.0/lib:/pkg/suse12/software/cURL/7.66.0-GCCcore-8.3.0/lib:/pkg/suse12/software/HDF5/1.10.5-gompic-2019b/lib:/pkg/suse12/software/Szip/2.1.1-GCCcore-8.3.0/lib:/pkg/suse12/software/SciPy-bundle/2019.10-fosscuda-2019b-Python-3.7.4/lib/python3.7/site-packages/numpy/core/lib:/pkg/suse12/software/SciPy-bundle/2019.10-fosscuda-2019b-Python-3.7.4/lib:/pkg/suse12/software/python/3.7.4-gcccore-8.3.0/lib:/pkg/suse12/software/libffi/3.2.1-gcccore-8.3.0/lib64:/pkg/suse12/software/libffi/3.2.1-gcccore-8.3.0/lib:/pkg/suse12/software/gmp/6.1.2-gcccore-8.3.0/lib:/pkg/suse12/software/sqlite/3.29.0-gcccore-8.3.0/lib:/pkg/suse12/software/tcl/8.6.9-gcccore-8.3.0/lib:/pkg/suse12/software/libreadline/8.0-gcccore-8.3.0/lib:/pkg/suse12/software/ncurses/6.1-gcccore-8.3.0/lib:/pkg/suse12/software/bzip2/1.0.8-gcccore-8.3.0/lib:/pkg/suse12/software/NCCL/2.4.8-gcccuda-2019b/lib:/pkg/suse12/software/cuDNN/7.6.4.38-gcccuda-2019b/lib64:/pkg/suse12/software/ScaLAPACK/2.0.2-gompic-2019b/lib:/pkg/suse12/software/FFTW/3.3.8-gompic-2019b/lib:/pkg/suse12/software/OpenBLAS/0.3.7-GCC-8.3.0/lib:/pkg/suse12/software/OpenMPI/3.1.4-gcccuda-2019b/lib:/pkg/suse12/software/hwloc/1.11.12-GCCcore-8.3.0/lib:/pkg/suse12/software/libpciaccess/0.14-GCCcore-8.3.0/lib:/pkg/suse12/software/libxml2/2.9.9-GCCcore-8.3.0/lib:/pkg/suse12/software/xz/5.2.4-gcccore-8.3.0/lib:/pkg/suse12/software/numactl/2.0.12-GCCcore-8.3.0/lib:/pkg/suse12/software/CUDA/10.1.243-GCC-8.3.0/nvvm/lib64:/pkg/suse12/software/CUDA/10.1.243-GCC-8.3.0/extras/CUPTI/lib64:/pkg/suse12/software/CUDA/10.1.243-GCC-8.3.0/lib64:/pkg/suse12/software/binutils/2.32-gcccore-8.3.0/lib:/pkg/suse12/software/zlib/1.2.11-gcccore-8.3.0/lib:/pkg/suse12/software/gcccore/8.3.0/lib/gcc/x86_64-pc-linux-gnu/8.3.0:/pkg/suse12/software/gcccore/8.3.0/lib64:/pkg/suse12/software/gcccore/8.3.0/lib
2020-11-11 11:07:00.528988: W tensorflow/stream_executor/cuda/cuda_driver.cc:312] failed call to cuInit: UNKNOWN ERROR (303)
2020-11-11 11:07:00.529008: I tensorflow/stream_executor/cuda/cuda_diagnostics.cc:156] kernel driver does not appear to be running on this host (cl4n023): /proc/driver/nvidia/version does not exist
2020-11-11 11:07:00.529515: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  AVX512F
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2020-11-11 11:07:00.533169: I tensorflow/core/common_runtime/process_util.cc:146] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
/var/spool/PBS/mom_priv/jobs/8413354.pbs.SC: line 28: 14317 Killed                  python lstm_test.py
